packages/client-core/src/common/components/Avatar/Avatar2.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import Button from '@ir-engine/ui/src/primitives/tailwind/Button'
+import Text from '@ir-engine/ui/src/primitives/tailwind/Text'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { FiEdit2 } from 'react-icons/fi'
+import { twMerge } from 'tailwind-merge'
+import { handleSoundEffect } from '../../utils'
+
+interface Props {
+  alt?: string
+  className?: string
+  id?: string
+  imageSrc?: string
+  isSelected?: boolean
+  name?: string
+  showChangeButton?: boolean
+  type?: 'round' | 'rectangle' | 'thumbnail'
+  size?: number
+  onChange?: () => void
+  onClick?: () => void
+}
+
+const Avatar = ({
+  alt,
+  className,
```

Comment:
the `Avatar` component is used just once and mostly like the same variant will be used - we can directly use the `className` values where required instead of passing them as props